Documentation/gettingstarted/hubble_cli.rst

```diff
@@ -33,7 +37,7 @@ In order to generate some network traffic, run the connectivity test:
 
 .. code-block:: shell-session
 
-   while true; do cilium connectivity test; done 
+   while true; do cilium connectivity test; done
```

Comment:
I think this whole subsection should be gone. The Star Wars demo does not rely on the connectivity tests.